### ISSUE TITLE
[CLI]: `wandb sync` does

### ISSUE THREAD

**Issue by rjbruin** [wandb/wandb#4420]
### Describe the bug

```python

```

```shell

```

### Environment

WandB version:

OS:

Python version:

Versions of relevant libraries:

**Comment by MBakirWB** [comment]
Hi @rjbruin , happy to help. Could you please expand on this post. What did you mean by the summaries of the run are stuck on the last value? Are you saying that in the [run overview page](https://wandb.ai/tudcv/vit_position_embeddings?workspace=user-mohammadbakir), the Summary isn't updating correctly post sync?

**Comment by rjbruin** [comment]
Exactly. For example, the value of `epoch` gets stuck at something like 15 even though I ran for 200 epochs. Similarly, the value for `val/acc` in the table reports the value for the 15th epoch, instead of for the final epoch. In the graphs, I can see the values for all the epochs. When I do CSV export, it shows the value from the table (the stuck value) rather than the last value from the graph.

**Comment by MBakirWB** [comment]
Hi @rjbruin , could you please provide me the entire run folder of a run that wasn't synching completely. I will attempt to reproduce this behavior on my end and review your logs.

**Comment by rjbruin** [comment]
Hi @MBakirWB, I've sent the requested info by email!

**Comment by MBakirWB** [comment]
Hi @rjbruin, I appreciate your patience while we looked into this. I was able to reproduce this issue frequently and have flagged it as a bug. It's now on the engineering teams radar. Once there's been movement or any updates, I'll be sure to let you know.

**Comment by rjbruin** [comment]
Excellent! Thanks for following up on this so thoroughly. :)

**Comment by RobertKirk** [comment]
I'm seeing a similar issue with my runs (I think) - we're running entirely offline, and then when we call wandb sync the summaries for the runs we've synced are empty. This makes it quite difficult to use the web ui to make bar charts, or other plots that rely on the summary metric rather than the logged metrics. Is there any update on this?

**Comment by wilrop** [comment]
I am having similar problems. I run my jobs offline as well and later sync my runs (like the person above me). As a potential workaround, is there a way to add new (key, values) to a completed summary after syncing? If yes, I can at least add certain stats to my summary which will help in comparing runs with different hyperparameters.

**Comment by MBakirWB** [comment]
Hi @wilrop , the issue is currently in the sdk teams backlog and will be revisited at a later time. You could update the summary metrics using the api, please visit [this page in our docs](https://docs.wandb.ai/guides/track/public-api-guide#update-summary-metrics) for an example,

**Comment by ArtsiomWB** [comment]
Hey guys! I was notified by my SDK team that this behavior should now be resolved. Please let me know if you're seeing something else on your end.
